pkg/terminal/command.go

```diff
@@ -604,25 +625,34 @@ func goroutines(t *Term, ctx callContext, argstr string) error {
 	if err != nil {
 		return err
 	}
-	gs, err := t.client.ListGoroutines()
-	if err != nil {
-		return err
-	}
-	sort.Sort(byGoroutineID(gs))
-	fmt.Printf("[%d goroutines]\n", len(gs))
-	for _, g := range gs {
-		prefix := "  "
-		if state.SelectedGoroutine != nil && g.ID == state.SelectedGoroutine.ID {
-			prefix = "* "
+	start := 0
+	gslen := 0
+	count := goroutineBatchSize
+	for start >= 0 {
+		gs, nextg, err := t.client.ListGoroutines(start, count)
+		if err != nil {
+			return err
 		}
-		fmt.Printf("%sGoroutine %s\n", prefix, formatGoroutine(g, fgl))
-		if bPrintStack {
-			stack, err := t.client.Stacktrace(g.ID, 10, false, nil)
+		if start == 0 && nextg == -1 {
+			sort.Sort(byGoroutineID(gs))
+			fmt.Printf("[%d goroutines]\n", len(gs))
```

Comment:
We are not beholden by backwards compatibility to keep this output unchanged so let's write the number of goroutines at the end of the end regardless. Also sort every time.